docs/src/main/sphinx/functions/table.md

```diff
@@ -46,10 +46,7 @@ The argument `columns` is a descriptor without types.
 
 (sequence-table-function)=
 :::{function} sequence(start => bigint, stop => bigint, step => bigint) -> table(sequential_number bigint)
-:noindex: true
```

Comment:
@mosabua Do you remember why the index is disabled here?

Reply:
@mosabua Gentle reminder.

Reply:
It used to fail the build since there is another function with the same name in array.md .. it doesnt fail now however.